### ISSUE TITLE
login form

### ISSUE THREAD

**Issue by Ashasindhu** [KiranAminPanjwani/LeatherHoard#14]
we can create a login form with email and password.can I work on this.

**Comment by KiranAminPanjwani** [comment]
Yeah sure, please go ahead!🤩 Make sure the layout/design of the login form complements the overall design of the website.😊

**Comment by Kpallavi5** [comment]
Hyy i want to work on this issue.. Can you please assign me..?
